apps/cli/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import { parseArgs } from "node:util";
+import { GLOBAL_CONFIG_FILE } from "./constants";
+import { ensureFileInPath } from "./utils";
+import { link } from "./link";
+import { showHelp, CLI_ARGS_OPTIONS, Commands } from "./args";
+import type { SupportedCommands } from "./args";
+import packageJSON from "../package.json" assert { type: "json" };
+
+const commands: SupportedCommands = {
+  link,
+};
+
+export const main = async () => {
+  try {
+    await ensureFileInPath(GLOBAL_CONFIG_FILE, JSON.stringify({}));
+    const args = parseArgs({
+      args: process.argv.slice(2),
+      options: CLI_ARGS_OPTIONS,
+      allowPositionals: true,
+    });
+
+    if (args.values?.version) {
+      console.log(packageJSON.version);
```

Comment:
same, lets switch to console.info everywhere and also add a linter against console.log, separate pr ofc